haystack/__init__.py

```diff
@@ -11,12 +11,6 @@
 __version__ = (2, 1, 1, 'dev')
 
 
```

Comment:
nit: Please remove the extra line... otherwise we have 3 blank lines and a linter will complain